### PR TITLE
Fetch weather data

### PR DESCRIPTION
**Pull request by MassterKek**
Still not functioning as intended

### CODE REVIEW

**Comment by cronin4392** [comment]
What issues are you running into? Does something come out of the `console.log`?

File: js/site.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
 // fetch weather data
 
+  let lat;
```
Comment (cronin4392) [inline js/site.js]: I think one issue you're gonna have it `lat` and `long` are not set. In the demo I did with the teens we used the browser tool to get your current position. To start though you can just hard code lat and long. I've done this by going on google maps and searching a location. When you do so the URL should update to something like:

```
https://www.google.com/maps/place/Artists+For+Humanity/@42.3429718,-71.0557969,17z/data=!3m2!4b1!5s0x89e37a7cc30fd4d5:0xa53c3af1827a4aa9!4m5!3m4!1s0x89e37a7cc503b901:0xa35db6dcf24f827e!8m2!3d42.3429718!4d-71.0536082
```

The lat and long from that url is `42.3429718` and `-71.0557969`. Please check the weather api documentation for the format they want the lat and long in.

Comment (cronin4392) [reply js/site.js]: Their docs are [here](https://www.weatherapi.com/docs/).

Comment (MassterKek) [reply js/site.js]: I am now getting an object displayed in the console. I assume I should interpret the data from that object and adjust the circles based on that data?

**Comment by cronin4392** [comment]
Looking good @MassterKek. For this PR I'd hook up the color to the current temp. Also, you could test by inputting hard coded temps to test a broad range since the temp for the day is gonna be constant. The final code will still use the current temp.

File: js/site.js [hunk 2]
```diff
@@ -1,18 +1,66 @@
...
+    const { text, icon } = data.current.condition;
+
+    const {
+      maxtemp_f,
+      mintemp_f,
+      maxtemp_c,
+      mintemp_c,
+    } = data.forecast.forecastday[0].day;
+
+    const hour_0_temp_f = data.forecast.forecastday[0].hour[0].temp_f;
```
Comment (cronin4392) [inline js/site.js]: I'd remove variables that aren't being used. I think VS Code highlights them?

**Review by cronin4392 — APPROVED** [review]

**Pull request merged**